EliteDangerous/DB/EDCommander.cs

```diff
@@ -136,19 +136,19 @@ public static void Delete(EDCommander cmdr)
 
         public static EDCommander Create(EDCommander other )
         {
-            return Create(other.name, other.EdsmName, other.EDSMAPIKey, other.JournalDir, other.syncToEdsm, other.SyncFromEdsm, 
-                                other.SyncToEddn, other.SyncToEGO, other.EGOName, other.EGOAPIKey, other.SyncToInara, other.InaraName, other.InaraAPIKey);
+            return Create(other.name, other.EdsmName, other.EDSMAPIKey, other.JournalDir, other.syncToEdsm, other.SyncFromEdsm,
+                                other.SyncToEddn, other.SyncToEGO, other.EGOName, other.EGOAPIKey, other.SyncToInara, other.InaraName, other.InaraAPIKey, other.SyncToIGAU);
```

Comment:
```
DB\EDCommander.cs(140,155): error CS1503: Argument 14: cannot convert from 'bool' to 'string' [C:\projects\eddiscovery\EliteDangerous\EliteDangerous.csproj]
```

Try:
```
... , toigau: other.SyncToIGAU
```

Reply:
I'm not quite sure what you mean.  Should this be line 139/140?

            return Create(other.name, other.EdsmName, other.EDSMAPIKey, other.JournalDir, other.syncToEdsm, other.SyncFromEdsm,
                                other.SyncToEddn, other.SyncToEGO, other.EGOName, other.EGOAPIKey, other.SyncToInara, other.InaraName, other.InaraAPIKey, toigau: other.SyncToIGAU);

I was looking at how EGO and EDSM were being handled.

Reply:
> Should this be line 139/140?
> ```
>         return Create(other.name, other.EdsmName, other.EDSMAPIKey, other.JournalDir, other.syncToEdsm, other.SyncFromEdsm,
>                             other.SyncToEddn, other.SyncToEGO, other.EGOName, other.EGOAPIKey, other.SyncToInara, other.InaraName, other.InaraAPIKey, toigau: other.SyncToIGAU);
> ```

Yes.

The parameters after `toinara` are:
```
string inaraname = null,
string inaraapikey = null,
string homesystem = null,
float mapzoom = 1.0f,
bool mapcentreonselection = true,
int mapcolour = -1,
bool toigau = false
```

Therefore `other.SyncToIGAU` was mapping to `homesystem`, where you want to map it to `toigau`